src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -18,6 +18,7 @@ public interface INodeHandler : IHandler
 
     public sealed class NodeHandler : Handler, INodeHandler
     {
+        private const int _environmentVariableMaximumSize = 32768;
```

Comment:
isn't the limit 32760? 
https://superuser.com/questions/1070272/why-does-windows-have-a-limit-on-environment-variables-at-all

have you checked that node task can get env. variable of size 32768 but not 32769?

Reply:
The limit is 32766, I validated.